core/src/test/java/io/questdb/test/cutlass/pgwire/PGJobContextTest.java

```diff
@@ -1772,13 +1774,59 @@ public void testCairoException() throws Exception {
         });
     }
 
+    @Test
+    public void testCancelRunningCreateQuery() throws Exception {
+        String[] queries = {"create table new_tab as (select count(*) from tab t1 cross join tab t2 where t1.x > 0)",
+                "select count(*) from tab t1 cross join tab t2 where t1.x > 0",
+                "insert into dest select count(*)::timestamp, 0, 0.0 from tab t1 cross join tab t2 where t1.x > 0",
+                "update dest set l = 1 from tab t1 where " +
+                        "'AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA' || t1.x = " +
+                        "'AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA100000' "
```

Comment:
Can we have a test where NetworkSqlExecutionCircuitBreaker is resued successfully?
And a test with multiple queries running, one is canceled and as the result, only the requested query is canceled.

Reply:
Added these tests (with cb reuse in both the same and separate connection) in fd45433e  .

Reply:
Done in latest commit .